H/HepMC3/build_tarballs.jl

```diff
@@ -0,0 +1,49 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilderBase, BinaryBuilder, Pkg
+
+name = "HepMC3"
+version = v"3.2.2"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("http://hepmc.web.cern.ch/hepmc/releases/HepMC3-3.2.2.tar.gz", "0e8cb4f78f804e38f7d29875db66f65e4c77896749d723548cc70fb7965e2d41")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/HepMC3-*/
+mkdir build && cd build/
+cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release -DHEPMC3_ENABLE_ROOTIO:BOOL=OFF -DHEPMC3_ENABLE_TEST:BOOL=ON -DHEPMC3_INSTALL_INTERFACES:BOOL=ON -DHEPMC3_ENABLE_PYTHON:BOOL=OFF -DHEPMC3_BUILD_STATIC_LIBS:BOOL=OFF -DHEPMC3_BUILD_DOCS:BOOL=OFF .. 
+make -j${nproc}
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Platform("i686", "linux"; libc="glibc", cxxstring_abi="cxx11"),
+    Platform("x86_64", "linux"; libc="glibc", cxxstring_abi="cxx11"),
+    Platform("aarch64", "linux"; libc="glibc", cxxstring_abi="cxx11"),
+    Platform("armv7l", "linux"; libc="glibc", cxxstring_abi="cxx11"),
+    Platform("powerpc64le", "linux"; libc="glibc", cxxstring_abi="cxx11"),
+    Platform("i686", "linux"; libc="musl", cxxstring_abi="cxx11"),
+    Platform("aarch64", "linux"; libc="musl", cxxstring_abi="cxx11"),
+    Platform("armv7l", "linux"; libc="musl", cxxstring_abi="cxx11"),
+    Platform("x86_64", "linux"; libc="musl", cxxstring_abi="cxx11"),
+    Platform("x86_64", "macos"; cxxstring_abi="cxx11")
```

Comment:
Why only C++11 string ABI?  For macOS probably it doesn't matter at all

Reply:
The website states as requirements: https://github.com/hep-mirrors/hepmc3
The minimal requirements for the installation from sources are
C++ compiler with c++11 standard support
CMake version 3.X. The support for version 2.8 is provided only for the core and search libraries. It will be completely removed in the future versions.

Reply:
I figured this means we don't need to worry about the old API, so we don't need to expand versions. Am I misunderstanding this?

Reply:
The C++ standard doesn't have anything to do with the version of C++ string ABI, that's a different matter: https://juliapackaging.github.io/BinaryBuilder.jl/dev/building/#Expanding-C-string-ABIs-or-libgfortran-versions.  So remove the `cxxstring_abi` everywhere here, but you'll need to use `expand_cxxstring_abis`